### ISSUE TITLE
Problem in "combined mode"

### ISSUE THREAD

**Issue by hartenthaler** [Neriderc/GVExport#141]
There is still a problem in the combined mode showing families without a corresponding box (F32, F1395).
![Screenshot (231)](https://user-images.githubusercontent.com/6324175/185813923-b5218a65-4e17-40df-92c2-550b1575d023.png)

When switching to simple mode (without changing anything else) everything looks perfect.
![Screenshot (234)](https://user-images.githubusercontent.com/6324175/185813889-ca0dd1b5-4bb3-443a-bc48-7224700ff8bf.png)

**Comment by Neriderc** [comment]
Do you know what's special about this scenario? Is it a particular relationship that is a problem?

I'm just wondering how I can reproduce the issue so I can look at fixing it.

**Comment by hartenthaler** [comment]
Another scenario for issue #100 .

Yes it is a bit special. I extracted the relevant GEDCOM code from my tree (remove the extension .txt!).
[hh_test.ged.txt](https://github.com/Neriderc/GVExport/files/9393347/hh_test.ged.txt)

Then set as starting persons X93 and I3177. As stopping person: I61. Select only "partner", nothing else. Max generation can be set to 6.
Then you should see:
![gvexport](https://user-images.githubusercontent.com/6324175/185897166-8f601e9d-c688-4aeb-8bce-9095cf887c7b.png)

In simple mode it is ok, in the combined mode it is strange.

**Comment by Neriderc** [comment]
I have pushed a commit to master that resolves this.

Basically, the stop person was detected and so it stops adding more. However, for combined mode, this leaves us with some interesting results as we end up with both of Kaspar's families but a missing family record. if we remove Kaspar or his family links then we don't get an expected result. In this case I have added a scan for "ghost" family records, where there is a family record in our list but it doesn't have anything of note (no spouse, no kids). These are then removed. In the case of a person without a spouse (in this case we have the example Theresia), a dummy family record is created by GVExport as part of the existing process, so I have ignored dummy records. Outside of dummy records all family records should have a spouse or children, so if they don't we remove the family record.

Here's the result I've got:
![image](https://user-images.githubusercontent.com/89974636/186276939-aacbc7da-67ea-4705-b6af-97371a1babd1.png)
